instrumentation/spring/spring-webflux-5.0/library/src/main/java/io/opentelemetry/instrumentation/spring/webflux/client/SpringWebfluxHttpAttributesGetter.java

```diff
@@ -56,6 +67,7 @@ public List<String> requestHeader(ClientRequest request, String name) {
   }
 
   @Override
+  @Nullable
   public Integer statusCode(
```

Comment:
I encountered the same issue in #7551 -- and decided not to use the `rawStatusCode()` method, it's already deprecated and a target for removal.

Reply:
I kept `rawStatusCode()` for version range `[5.1,6.0)`, but added new path for 6.0